INSTALL.md

```diff
@@ -1,95 +1,508 @@
-	git clone https://github.com/pret/agbcc
+2. Once the process finishes, restart your machine.
 
-	cd ./agbcc
-	./build.sh
-	./install.sh ../pokeemerald
+3. The next step is to choose and install a Linux distribution from the Microsoft Store. The following instructions will assume Ubuntu as the Linux distribution of choice.
 
-	cd ../pokeemerald
+    > Note for advanced users: Advanced users can pick a preferred Linux distribution, but setup instructions may differ. Debian should work with the given instructions, but has not been tested. 
 
-To build **pokeemerald.gba** for the first time and confirm it matches the official ROM image:
+4. Open the [Microsoft Store Linux Selection](https://aka.ms/wslstore), click Ubuntu, then click Get, which will install the Ubuntu distribution.
 
-	make compare
+    > Note 1: If a dialog pops up asking for you to sign into a Microsoft Account, then just close the dialog.  
+    > Note 2: If the link does not work, then open the Microsoft Store manually, and search for the Ubuntu app (choose the one with no version number).  
 
-If an OK is returned, then the installation went smoothly.
+### Setting up WSL1
+Some tips before proceeding:
+- In WSL, Copy and Paste is either done via
+    - **right-click** (selection + right click to Copy, right click with no selection to Paste)
+    - **Ctrl+Shift+C/Ctrl+Shift+V** (enabled by right-clicking the title bar, going to Properties, then checking the checkbox next to "Use Ctrl+Shift+C/V as Copy/Paste").
+- Some of the commands that you'll run will ask for your WSL password and/or confirmation to perform the stated action. This is to be expected, just enter your WSL password and/or the yes action when necessary.
 
-**Windows users:** Consider adding exceptions for the `pokeemerald` and `agbcc` folders in Windows Security using [these instructions](https://support.microsoft.com/help/4028485). This prevents Microsoft Defender from scanning them which might improve performance while building.
+1. Open **Ubuntu** (e.g. using Search).
+2. WSL/Ubuntu will set up its own installation when it runs for the first time. Once WSL/Ubuntu finishes installing, it will ask for a username and password (to be input in).
 
+    > Note: When typing in the password, there will be no visible response, but the terminal will still read in input.
 
-# Start
+3. Update WSL/Ubuntu before continuing. Do this by running the following command:
 
-To build **pokeemerald.gba** with your changes:
+    ```bash
+    sudo apt update && sudo apt upgrade
+    ```
+    > Note: These commands will likely take a long time to finish.
 
-	make
+> *Note: If the repository you plan to build was created before 2020/XX/YY (e.g. modifications of pokeemerald that haven't updated) then follow the [legacy WSL1 instructions](docs/legacy_WSL1_INSTALL.md). These repositories can be identified by the [older revision](https://github.com/pret/pokeemerald/blob/571c598/INSTALL.md) of the INSTALL.md*
```

Comment:
`was created before 2020/XX/YY (e.g. modifications of pokeemerald that haven't updated)`
How about `has not been updated since 2020/XX/YY` (also, replacing the date of merge when done. note its 2021 now)

Reply:
Maybe I'll just drop the date and make a note about older repositories in general. I'll think of something later. Saying "updated" might be misleading for modifications of pokeemerald which update past the date, but don't sync with pokeemerald.